src/themes/mainTheme.ts

```diff
@@ -138,32 +139,42 @@ export const mainTheme = createTheme(
           },
         },
       },
+      MuiStepIcon: {
+        styleOverrides: {
+          root: {
+            fill: Color.StepperBlue,
+            opacity: 0.6,
+            '&.Mui-active': {
+              opacity: 1,
+            },
+            '&.Mui-error': {
+              fill: Color.ErrorMain,
+            },
+          },
+        },
+      },
       MuiStepLabel: {
         styleOverrides: {
           label: {
             opacity: 0.6,
             textTransform: 'uppercase',
             fontSize: '1rem',
-            color: Color.PrimaryMain,
+            color: Color.StepperBlue,
             '&.Mui-active': {
-              color: Color.PrimaryMain,
+              color: Color.StepperBlue,
               fontWeight: 600,
+              borderBottom: '0.06rem solid',
+              mb: '-0.06rem',
+              boxShadow: '-1px 7px 4px -3px rgba(161,161,161,0.49)',
```

Comment:
Mye "greier" som skjer her som ikke er åpenbart for meg iaf 😅 
Åpent spørsmål: er det mulig å forenkle til noe som ikke virker så spesifikt og "låst"?

Reply:
Feks `rgba(0,0,0,0.2)`

Reply:
etter litt testing i cssgeneratoren får vi kanskje ikke noen gevinst av å bruke rem nei, så kan sikkert droppe det

Reply:
men syns fortsatt det virker litt ryddigere å bare bruke svart farge, som er mer transparent da